hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitInfosComponent.js

```diff
@@ -307,13 +361,31 @@ const OrgUnitInfosComponent = ({
         initialFormState(orgUnit, instanceId),
     );
 
+    const [isSaveDisabled, setSaveDisabled] = useState(true);
+
+    const showSpeedDialInstanceActions =
+        orgUnit.reference_instance ||
+        (formId === referenceFormId &&
+            formId !== undefined &&
```

Comment:
`Boolean(formId)  && Boolean(referenceFormId)`, it will work if formId and referenceFormId are null